YOLO.xcodeproj/project.pbxproj

```diff
@@ -210,13 +195,8 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				6381D21B2B7817C200ABA4E8 /* yolov8m.mlpackage in Sources */,
-				6381D21C2B7817C200ABA4E8 /* yolov8n.mlpackage in Sources */,
 				636EFCAF21E62DD300DE43BC /* VideoCapture.swift in Sources */,
 				636166EA251443B20054FA7E /* ThresholdProvider.swift in Sources */,
-				6381D2182B7817C200ABA4E8 /* yolov8l.mlpackage in Sources */,
```

Comment:
Again, we don't want to eliminate these references!

Reply:
I changed those lines back and restored the model references.